tests/src/test/resources/test-cases/wireRec.success

```diff
@@ -0,0 +1,15 @@
+case class A()
+case class B()
+case class C(a: A, b: B)
+case class D(a: A, c: C)
+
+object Test {
```

Comment:
you could also add another test, with a two (or more) step process of recursive wiring. E.g. the same as here, but without `a` and `b` explicitly wired